### PR TITLE
VACMS-15339: Mobile promo banner

### PR DESCRIPTION
**Pull request by chriskim2311**
## Description
Initial implementation of Smartbanner.js. New implementation will impact all android devices and non-safari iOS browsers. There will be app store specific promo banners and promo banners for just specific URLs listed below.

      '/health-care/refill-track-prescriptions',
      '/health-care/secure-messaging',
      '/health-care/get-medical-records',
      '/disability/view-disability-rating',
      '/claim-or-appeal-status',
      '/disability/upload-supporting-evidence',
      '/records/download-va-letters',
      '/va-payment-history',
      '/change-direct-deposit',

Relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15318

## Testing done & Screenshots
Testing was done for the URLs that the promo will be valid for, through ngrok and browserstack. 

Android App Store View
Samsung_Galaxy_S22_Ultra_v12_0

Non-Safari iOS Browser App Store View

iPhone_13_v15_4

### CODE REVIEW

File: src/site/includes/header.html [hunk 1]
```diff
@@ -188,3 +193,13 @@
   <script nonce="**CSP_NONCE**" type="text/javascript">
     {% include "src/site/assets/js/skip-link-focus.js" %}
   </script>
+
+  <script  type="text/javascript" defer>
+    window.onload = function() {
+      const bodyDiv =  document.querySelector('body');
```
[the remark this answers is not in the document]
Comment (randimays) [reply src/site/includes/header.html:198]: It doesn't seem like `bodyDiv` is being used. Is it needed here?

File: src/site/includes/header.html [hunk 2]
```diff
@@ -188,3 +192,13 @@
...
+      const contentElement = document.querySelector('.smartbanner');
+      if(contentElement){
```
Comment (randimays) [inline src/site/includes/header.html:200]: There are some spacing / indentation issues here

**Review by randimays — COMMENTED** [review]
I'm not sure if this is something we can control, or something we want to match, but the versions here have a CTA that says "View," and the [iOS Safari](https://github.com/department-of-veterans-affairs/content-build/pull/1632) version says "Get." Do we need/want to update?

File: src/site/includes/header.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (chriskim2311) [reply src/site/includes/header.html:198]: Yup going to remove this.

**Comment by chriskim2311** [comment]
Not sure I can change the text from View to Get unfortunately.

**Review by randimays — APPROVED** [review]
LGTM!

**Review by pjhill — APPROVED** [review]

**Pull request merged**